extensions/wrapper/wrapper-common-api/src/main/java/de/sovity/edc/ext/wrapper/api/common/model/UiAssetDistribution.java

```diff
@@ -0,0 +1,29 @@
+package de.sovity.edc.ext.wrapper.api.common.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@RequiredArgsConstructor
+@Schema(description = "Asset Distribution  Details")
+public class UiAssetDistribution {
+
+    @JsonProperty("@type")
+    @Schema(description = "Type", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String type;
+
+    @JsonProperty("http://www.w3.org/ns/dcat#mediaType")
+    @Schema(description = "Media Type", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String name;
+}
```

Comment:
this class does not belong in `wrapper-common-api` with its `@JsonProperty` fields